repository/src/main/java/org/apache/atlas/util/RepairIndex.java

```diff
@@ -81,7 +90,8 @@ private static void reindexVertex(String indexName, IndexSerializer indexSeriali
                 }
             }
         }
-        mutator.getIndexTransaction(indexType.getBackingIndexName()).restore(documentsPerStore);
+        //mutator.getIndexTransaction(indexType.getBackingIndexName()).restore(documentsPerStore);
+        searchIndex.restore(documentsPerStore, indexSerializer.getIndexInfoRetriever(tx).get("search"));
```

Comment:
Does it reindex the tag attributes?? I do not see any logic to fetch tags for asset & generate tag attributes for asset in flow
Pleas test & let me know